<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Helicopter_WeaponWindow</class>
 <widget class="QWidget" name="Helicopter_WeaponWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>743</width>
    <height>306</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Helicopter_WeaponWindow</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>734</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="grid">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item row="2" column="5">
     <widget class="QLabel" name="bd6">
      <property name="text">
       <string>БД6 
 ЗАГРУЖ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="bd1">
      <property name="text">
       <string>БД1 
 ЗАГРУЖ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>КОРОТКАЯ         ДЛИННАЯ          СРЕДНЯЯ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QCheckBox" name="t_main_switch">
      <property name="text">
       <string>ГЛАВНЫЙ
 ВЫКЛЮЧАТЬЕЛЬ</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="ab">
      <property name="text">
       <string>АБ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="bd2">
      <property name="text">
       <string>БД2 
 ЗАГРУЖ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="4" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ВНЕШНЯЯ                ВСЕ                   ВНУТР</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="npc">
      <property name="text">
       <string>НРС</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>FAB             NAR            UPK23          PKT</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="bd3">
      <property name="text">
       <string>БД3 
 ЗАГРУЖ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QLabel" name="pus25">
      <property name="text">
       <string>ПУС ВЗВЕД
БД2 БД5</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLCDNumber" name="lcdNumber"/>
    </item>
    <item row="3" column="2">
     <widget class="QCheckBox" name="t_pus_load">
      <property name="text">
       <string>ВЗВЕДЕНИЕ
ПУС</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="vzriv">
      <property name="text">
       <string>ВЗРЫВ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLCDNumber" name="lcdNumber_2"/>
    </item>
    <item row="3" column="4">
     <widget class="QCheckBox" name="t_vzriv">
      <property name="text">
       <string>ВЗРЫВ
НЕ ВЗРЫВ</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="2">
     <widget class="QSlider" name="sw_burst">
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="pus16">
      <property name="text">
       <string>ПУС ВЗВЕД
БД1 БД6</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="cepi">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>цепи
 вооружения
 отключены</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="4" colspan="2">
     <widget class="QSlider" name="sw_pos">
      <property name="maximum">
       <number>2</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="bd5">
      <property name="text">
       <string>БД5 
 ЗАГРУЖ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QSlider" name="sw_weapon">
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>3</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::TicksAbove</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QLabel" name="upk">
      <property name="text">
       <string>УПК</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="5">
     <widget class="QCheckBox" name="t_sbros">
      <property name="text">
       <string>АВАРИЙНЫЙ
СБРОС</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="bd4">
      <property name="text">
       <string>БД4 
 ЗАГРУЖ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="6">
     <widget class="QSlider" name="PKT_horizont">
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="6" rowspan="7">
     <widget class="QSlider" name="PKT_vert">
      <property name="value">
       <number>50</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="7" column="6">
     <widget class="QPushButton" name="pkt_fire">
      <property name="text">
       <string>F</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="0">
     <widget class="QCheckBox" name="t_safety_lock">
      <property name="text">
       <string>Предохранитель</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="i_weapon_warning">
      <property name="text">
       <string>ОСТОРОЖНО
ЦЕПИ ВООРУЖЕНИЯ
ПОД ТОКОМ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
